src/microbe_stage/Membrane.cs

```diff
@@ -364,8 +364,10 @@ private void ApplyTint()
 
     private void ApplyTextures()
     {
-        if (normalTexture != null)
-            return;
+        // We must update the texture on already-existing membranes,
+        // due to the membrane texture changing for the player microbe.
+        // if (normalTexture != null)
+        //     return;
```

Comment:
I think that instead of removing this check instead what could be done is adding a new string field called `currentlyLoadedNormalTexture` which would have the texture path.
And then this check could be `if(normalTexture != null && currentlyLoadedNormalTexture == Type.NormalTexture)`